content/source/docs/enterprise/sentinel/import/tfconfig.html.md

```diff
@@ -132,8 +132,8 @@ import "tfconfig"
 filename_value = tfconfig.resources.local_file.accounts.config.filename
 
 main = rule {
-	filename_value contains "${var.domain}" and
-	filename_value contains "${var.subdomain}"
+  filename_value contains "${var.domain}" and
```

Comment:
Can you restore the tabs here? `sentinel fmt` currently uses tabs and that's the way examples should stay until this changes.

This should be the same way for all of the other examples as well.

Reply:
I'll fix the formatting.